Src/coffeescript/cql-execution/src/datatypes/interval.coffee

```diff
@@ -7,60 +7,60 @@ cmp = require '../util/comparison'
 module.exports.Interval = class Interval
   constructor: (@low, @high, @lowClosed = true, @highClosed = true) ->
 
-  contains: (item) ->
+  contains: (item, precision) ->
     if item instanceof Interval then throw new Error("Argument to contains must be a point")
     closed = @toClosed()
     ThreeValuedLogic.and(
-      cmp.lessThanOrEquals(closed.low, item),
-      cmp.greaterThanOrEquals(closed.high, item)
+      cmp.lessThanOrEquals(closed.low, item, precision),
+      cmp.greaterThanOrEquals(closed.high, item, precision)
     )
 
-  properlyIncludes: (other) ->
+  properlyIncludes: (other, precision) ->
     if not (other instanceof Interval) then throw new Error("Argument to properlyIncludes must be an interval")
     ThreeValuedLogic.and(
-      @includes(other),
-      ThreeValuedLogic.not(cmp.equals(@, other))
```

Comment:
I don't understand the unit tests well enough, have we characterized the changes from changing from equals to includes?

Do we know why this hasn't caused issues previously?